skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/apps/AnnotatePlainTextFiles.scala

```diff
@@ -0,0 +1,64 @@
+package org.ml4ai.skema.text_reading.apps
+
+import org.clulab.utils.Logging
+import org.ml4ai.skema.text_reading.utils.{ArgsConfig, CommandLineArgumentParser}
+import org.ml4ai.skema.text_reading.PlainTextFileTextReadingPipeline
+
+import scala.util.Using
+import scopt.OParser
+
+import java.io.{File, FileOutputStream, PrintWriter}
+
+
+object AnnotatePlainTextFiles extends App with Logging{
+
+  val parser = CommandLineArgumentParser.buildParser("AnnotatePlainTextFiles")
```

Comment:
This could be future-proofed with `buildParser(getClass.getSimpleName.dropRight(1))` or the `CommandLineArgumentParser` could take some other kind of parameter and generate the name from it similarly.